rust/noosphere-api/Cargo.toml

```diff
@@ -20,6 +20,7 @@ readme = "README.md"
 
 [dependencies]
 anyhow = "^1"
+thiserror = "1.0.38"
```

Comment:
Should we use this going forward on decorating our own Errors?

Reply:
Yah, we have used it in two places already